mesh/ncmesh.cpp

```diff
@@ -2751,6 +3043,8 @@ int NCMesh::QuadFaceSplitType(int v1, int v2, int v3, int v4,
    if (e1 >= 0 && e3 >= 0) { midf1 = FindMidEdgeNode(e1, e3); }
    if (e2 >= 0 && e4 >= 0) { midf2 = FindMidEdgeNode(e2, e4); }
 
+   s = 0.5;
```

Comment:
This seems to only get used on the non-split branch, but if the face is not split, then surely the scale is supposed to be 0? Why not delete this and place `s=0.0` in the non-split branch below.

Reply:
This value is not used in the non-split case, so I set it to the invalid value 0 as you suggested in e5cfc35f87969a015c88f892f8927267bd9ef4c5